code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -63,7 +63,7 @@
 
 
 
-/obj/item/organ/body_egg/alien_embryo/proc/AttemptGrow(gib_on_success=TRUE)
+/obj/item/organ/body_egg/alien_embryo/proc/AttemptGrow(var/kill_on_sucess=TRUE)

```

Comment:
```suggestion
/obj/item/organ/body_egg/alien_embryo/proc/AttemptGrow(kill_on_sucess=TRUE)
```

Reply:
what does this do?

Reply:
the `var/` prefix is already implied and doesn't need to be stated.

As for the scope of this arg, I guess it has something to do with the edge cases of surgically removing a larva seconds before it'd violently burst out its victim themselves.

Reply:
Putting var before is not invalid and merely comes down to personal preference.

Reply:
it's redunant, but merely a tiny waste of space after all. Go on.